### DISCUSSION TITLE
what is the oject?

### DISCUSSION THREAD

**Discussion by Taxom** [jvde-github/AIS-catcher#121] [Q&A]
I receive base station(?) MMSI:3160029, previously it had location which always changes. Now I don't get anymore its location(always pending). 
I tried decoding messages from it in AIS Decoder 3.5.149. And it seems to me that these are reports about the weather in different places (so location is changing). This is the correct assumption, and is it possible to somehow display it on the AisCather map?

Thanks!
Max.

![kjlndfgvjqng jn aer ojgnmqwer](https://github.com/jvde-github/AIS-catcher/assets/20313019/e21c8b94-660a-4e72-8066-6d06d119a5b3)
![Screenshot 2023-06-16 at 20-40-55 AIS-catcher](https://github.com/jvde-github/AIS-catcher/assets/20313019/11f61fa1-6524-45f0-9e3a-a1563af1dcc8)

**Comment by jvde-github** [comment]
Hello, thanks for the observation. This was indeed something that has changed recently. The message is a weather update for the lat/lon. Previously this lat/lon was used as location for the MMSI. But as these messages are send by one MMSI for multiple locations, it was moving the marker all over the map and confusing users. That was not correct so we do not use the send lat/lon anymore as location for the MMSI sender.

Possibly we could create an artificial MMSI with a artifical shiptype and plot that on the map.  Will be a bit of work but can add it to the TO DO list. Not sure when I have time to look at it though.

Do you have a few NMEA lines from this MMSI for testing?

**Comment by Taxom** [comment]
Is this fine?

`!AIVDM,2,1,9,A,8@30oo@0GufGUQ?<QR=GwwvlFR06EuOwgwl?wnSwe7wvlOwwsAwwnSGm,0*51,17/06/2023 21:31:24
!AIVDM,2,2,9,A,wvh0,0*46,17/06/2023 21:31:24
!AIVDM,2,1,0,A,8@30oo@0GufGUQ?<QR=GwwvlFR06EuOwgwl?wnSwe7wvlOwwsAwwnSGm,0*58,17/06/2023 21:31:24
!AIVDM,2,2,0,A,wvh0,0*4F,17/06/2023 21:31:24
!AIVDM,2,1,3,A,8@30oo@0GufpSa>PEr=I?wvlFR06EuOwgr:wwnSwe7wvlOwwsAwwnSGm,0*5E,17/06/2023 21:38:26
!AIVDM,2,2,3,A,wvh0,0*4C,17/06/2023 21:38:26
!AIVDM,2,1,4,A,8@30oo@0GufpSa>PEr=I?wvlFR06EuOwgr:wwnSwe7wvlOwwsAwwnSGm,0*59,17/06/2023 21:38:26
!AIVDM,2,2,4,A,wvh0,0*4B,17/06/2023 21:38:26
!AIVDM,2,1,5,A,8@30oo@0GuftSA?:kr=I?wvlFR06EuOwgr:OwnSwe7wvlOwwsAwwnSGm,0*01,17/06/2023 21:38:26
!AIVDM,2,2,5,A,wvh0,0*4A,17/06/2023 21:38:26
!AIVDM,2,1,6,A,8@30oo@0GuftSA?:kr=I?wvlFR06EuOwgr:OwnSwe7wvlOwwsAwwnSGm,0*02,17/06/2023 21:38:26
!AIVDM,2,2,6,A,wvh0,0*49,17/06/2023 21:38:26
!AIVDM,2,1,7,A,8@30oo@0GufWD1?4S2=IgwvlFR06EuOwgwl?wnSwe7wvlOwwsAwwnSGm,0*4A,17/06/2023 21:38:31
!AIVDM,2,2,7,A,wvh0,0*48,17/06/2023 21:38:31
!AIVDM,2,1,3,A,8@30oo@0Gufpq1>OL2=KOwvlFR06EuOwgwl?wnSwe7wvlOwwsAwwnSGm,0*13,17/06/2023 21:45:57
!AIVDM,2,2,3,A,wvh0,0*4C,17/06/2023 21:45:57
!AIVDM,2,1,4,A,8@30oo@0Gufpq1>OL2=KOwvlFR06EuOwgwl?wnSwe7wvlOwwsAwwnSGm,0*14,17/06/2023 21:45:57
!AIVDM,2,2,4,A,wvh0,0*4B,17/06/2023 21:45:57
!AIVDM,2,1,5,A,8@30oo@0GufGUQ?<QR=KgwvlFR06EuOwgwl?wnSwe7wvlOwwsAwwnSGm,0*41,17/06/2023 21:46:26
!AIVDM,2,2,5,A,wvh0,0*4A,17/06/2023 21:46:26
!AIVDM,2,1,6,A,8@30oo@0GufGUQ?<QR=KgwvlFR06EuOwgwl?wnSwe7wvlOwwsAwwnSGm,0*42,17/06/2023 21:46:26
!AIVDM,2,2,6,A,wvh0,0*49,17/06/2023 21:46:26
!AIVDM,2,1,4,A,8@30oo@0GufpSa>PEr=LwwvlFR06EuOwgr:wwnSwe7wvlOwwsAwwnSGm,0*14,17/06/2023 21:53:29
!AIVDM,2,2,4,A,wvh0,0*4B,17/06/2023 21:53:29
!AIVDM,2,1,5,A,8@30oo@0GufpSa>PEr=LwwvlFR06EuOwgr:wwnSwe7wvlOwwsAwwnSGm,0*15,17/06/2023 21:53:29
!AIVDM,2,2,5,A,wvh0,0*4A,17/06/2023 21:53:29
!AIVDM,2,1,6,A,8@30oo@0GuftSA?:kr=LwwvlFR06EuOwgr:?wnSwe7wvlOwwsAwwnSGm,0*3F,17/06/2023 21:53:29
!AIVDM,2,2,6,A,wvh0,0*49,17/06/2023 21:53:29
!AIVDM,2,1,7,A,8@30oo@0GuftSA?:kr=LwwvlFR06EuOwgr:?wnSwe7wvlOwwsAwwnSGm,0*3E,17/06/2023 21:53:29
!AIVDM,2,2,7,A,wvh0,0*48,17/06/2023 21:53:30
!AIVDM,2,1,8,A,8@30oo@0GufWD1?4S2=MOwvlFR06EuOwgwl?wnSwe7wvlOwwsAwwnSGm,0*69,17/06/2023 21:53:33
!AIVDM,2,2,8,A,wvh0,0*47,17/06/2023 21:53:33
!AIVDM,2,1,9,A,8@30oo@0GufWD1?4S2=MOwvlFR06EuOwgwl?wnSwe7wvlOwwsAwwnSGm,0*68,17/06/2023 21:53:33
!AIVDM,2,2,9,A,wvh0,0*46,17/06/2023 21:53:33
!AIVDM,2,1,8,A,8@30oo@0GufGUQ?<QR=POwvlFR06EuOwgwl?wnSwe7wvlOwwsAwwnSGm,0*7F,17/06/2023 22:01:26
!AIVDM,2,2,8,A,wvh0,0*47,17/06/2023 22:01:26
!AIVDM,2,1,9,A,8@30oo@0GufGUQ?<QR=POwvlFR06EuOwgwl?wnSwe7wvlOwwsAwwnSGm,0*7E,17/06/2023 22:01:26
!AIVDM,2,2,9,A,wvh0,0*46,17/06/2023 22:01:26
!AIVDM,2,1,1,A,8@30oo@0GufpSa>PEr=QgwvlFR06EuOwgr:gwnSwe7wvlOwwsAwwnSGm,0*0C,17/06/2023 22:08:26
!AIVDM,2,2,1,A,wvh0,0*4E,17/06/2023 22:08:26
!AIVDM,2,1,2,A,8@30oo@0GufpSa>PEr=QgwvlFR06EuOwgr:gwnSwe7wvlOwwsAwwnSGm,0*0F,17/06/2023 22:08:26
!AIVDM,2,2,2,A,wvh0,0*4D,17/06/2023 22:08:26
!AIVDM,2,1,3,A,8@30oo@0GuftSA?:kr=QgwvlFR06EuOwgr:OwnSwe7wvlOwwsAwwnSGm,0*47,17/06/2023 22:08:26
!AIVDM,2,2,3,A,wvh0,0*4C,17/06/2023 22:08:26
!AIVDM,2,1,4,A,8@30oo@0GuftSA?:kr=QgwvlFR06EuOwgr:OwnSwe7wvlOwwsAwwnSGm,0*40,17/06/2023 22:08:28
!AIVDM,2,2,4,A,wvh0,0*4B,17/06/2023 22:08:28
!AIVDM,2,1,5,A,8@30oo@0GufWD1?4S2=R?wvlFR06EuOwgwl?wnSwe7wvlOwwsAwwnSGm,0*0B,17/06/2023 22:08:32
!AIVDM,2,2,5,A,wvh0,0*4A,17/06/2023 22:08:32
!AIVDM,2,1,6,A,8@30oo@0GufWD1?4S2=R?wvlFR06EuOwgwl?wnSwe7wvlOwwsAwwnSGm,0*08,17/06/2023 22:08:32
!AIVDM,2,2,6,A,wvh0,0*49,17/06/2023 22:08:32
!AIVDM,2,1,3,A,8@30oo@0GufpSa>PEr=UOwvlFR06EuOwgr:wwnSwe7wvlOwwsAwwnSGm,0*32,17/06/2023 22:23:24
!AIVDM,2,2,3,A,wvh0,0*4C,17/06/2023 22:23:24
!AIVDM,2,1,4,A,8@30oo@0GufpSa>PEr=UOwvlFR06EuOwgr:wwnSwe7wvlOwwsAwwnSGm,0*35,17/06/2023 22:23:24
!AIVDM,2,2,4,A,wvh0,0*4B,17/06/2023 22:23:24
!AIVDM,2,1,1,A,8@30oo@0GuftSA?:kr=UOwvlFR06EuOwgr:OwnSwe7wvlOwwsAwwnSGm,0*69,17/06/2023 22:23:25
!AIVDM,2,2,1,A,wvh0,0*4E,17/06/2023 22:23:25
!AIVDM,2,1,2,A,8@30oo@0GuftSA?:kr=UOwvlFR06EuOwgr:OwnSwe7wvlOwwsAwwnSGm,0*6A,17/06/2023 22:23:25
!AIVDM,2,2,2,A,wvh0,0*4D,17/06/2023 22:23:25
!AIVDM,2,1,3,A,8@30oo@0GufWD1?4S2=UwwvlFR06EuOwgwl?wnSwe7wvlOwwsAwwnSGm,0*42,17/06/2023 22:23:30
!AIVDM,2,2,3,A,wvh0,0*4C,17/06/2023 22:23:30
!AIVDM,2,1,4,A,8@30oo@0GufWD1?4S2=UwwvlFR06EuOwgwl?wnSwe7wvlOwwsAwwnSGm,0*45,17/06/2023 22:23:30
!AIVDM,2,2,4,A,wvh0,0*4B,17/06/2023 22:23:30
!AIVDM,2,1,8,A,8@30oo@0Gufpq1>OL2=WgwvlFR06EuOwgwl?wnSwe7wvlOwwsAwwnSGm,0*2C,17/06/2023 22:30:54
!AIVDM,2,2,8,A,wvh0,0*47,17/06/2023 22:30:54
!AIVDM,2,1,9,A,8@30oo@0Gufpq1>OL2=WgwvlFR06EuOwgwl?wnSwe7wvlOwwsAwwnSGm,0*2D,17/06/2023 22:30:54
!AIVDM,2,2,9,A,wvh0,0*46,17/06/2023 22:30:54
!AIVDM,2,1,8,A,8@30oo@0GufGUQ?<QR=WwwvlFR06EuOwgwl?wnSwe7wvlOwwsAwwnSGm,0*40,17/06/2023 22:31:25
!AIVDM,2,2,8,A,wvh0,0*47,17/06/2023 22:31:25
!AIVDM,2,1,9,A,8@30oo@0GufGUQ?<QR=WwwvlFR06EuOwgwl?wnSwe7wvlOwwsAwwnSGm,0*41,17/06/2023 22:31:25
!AIVDM,2,2,9,A,wvh0,0*46,17/06/2023 22:31:25
!AIVDM,2,1,4,A,8@30oo@0GufWD1?4S2=agwvlFR06EuOwgwl?wnSwe7wvlOwwsAwwnSGm,0*61,17/06/2023 22:38:31
!AIVDM,2,2,4,A,wvh0,0*4B,17/06/2023 22:38:31
!AIVDM,2,1,5,A,8@30oo@0GufWD1?4S2=agwvlFR06EuOwgwl?wnSwe7wvlOwwsAwwnSGm,0*60,17/06/2023 22:38:31
!AIVDM,2,2,5,A,wvh0,0*4A,17/06/2023 22:38:32
!AIVDM,2,1,5,A,8@30oo@0Gufpq1>OL2=cOwvlFR06EuOwgwl?wnSwe7wvlOwwsAwwnSGm,0*3D,17/06/2023 22:45:55
!AIVDM,2,2,5,A,wvh0,0*4A,17/06/2023 22:45:55
!AIVDM,2,1,6,A,8@30oo@0Gufpq1>OL2=cOwvlFR06EuOwgwl?wnSwe7wvlOwwsAwwnSGm,0*3E,17/06/2023 22:45:55
!AIVDM,2,2,6,A,wvh0,0*49,17/06/2023 22:45:55
!AIVDM,2,1,0,A,8@30oo@0GufGUQ?<QR=cgwvlFR06EuOwgwl?wnSwe7wvlOwwsAwwnSGm,0*6C,17/06/2023 22:46:27
!AIVDM,2,2,0,A,wvh0,0*4F,17/06/2023 22:46:27
!AIVDM,2,1,1,A,8@30oo@0GufGUQ?<QR=cgwvlFR06EuOwgwl?wnSwe7wvlOwwsAwwnSGm,0*6D,17/06/2023 22:46:27
!AIVDM,2,2,1,A,wvh0,0*4E,17/06/2023 22:46:27
!AIVDM,2,1,8,A,8@30oo@0GufpSa>PEr=dwwvlFR06EuOwgr:ownSwe7wvlOwwsAwwnSGm,0*28,17/06/2023 22:53:25
!AIVDM,2,2,8,A,wvh0,0*47,17/06/2023 22:53:25
!AIVDM,2,1,9,A,8@30oo@0GufpSa>PEr=dwwvlFR06EuOwgr:ownSwe7wvlOwwsAwwnSGm,0*29,17/06/2023 22:53:25
!AIVDM,2,2,9,A,wvh0,0*46,17/06/2023 22:53:25
!AIVDM,2,1,2,A,8@30oo@0GuftSA?:kr=dwwvlFR06EuOwgr:GwnSwe7wvlOwwsAwwnSGm,0*6B,17/06/2023 22:53:26
!AIVDM,2,2,2,A,wvh0,0*4D,17/06/2023 22:53:26
!AIVDM,2,1,3,A,8@30oo@0GuftSA?:kr=dwwvlFR06EuOwgr:GwnSwe7wvlOwwsAwwnSGm,0*6A,17/06/2023 22:53:26
!AIVDM,2,2,3,A,wvh0,0*4C,17/06/2023 22:53:26
!AIVDM,2,1,0,A,8@30oo@0Gufpq1>OL2=h?wvlFR06EuOwgwl?wnSwe7wvlOwwsAwwnSGm,0*43,17/06/2023 23:00:54
!AIVDM,2,2,0,A,wvh0,0*4F,17/06/2023 23:00:54
!AIVDM,2,1,1,A,8@30oo@0Gufpq1>OL2=h?wvlFR06EuOwgwl?wnSwe7wvlOwwsAwwnSGm,0*42,17/06/2023 23:00:54
!AIVDM,2,2,1,A,wvh0,0*4E,17/06/2023 23:00:54`

**Comment by Taxom** [comment]
[output_20230618.csv](https://github.com/jvde-github/AIS-catcher/files/11780845/output_20230618.csv)

**Comment by jvde-github** [comment]
This works, thank you. I need to give it some more thought how to embed in the webserver but have added it to the list.
